### PR TITLE
New Package Layout

### PR DESCRIPTION
**Pull request by pmuetschard**
See #152

### CODE REVIEW

File: core/app/layout/layout.go [hunk 1]
```diff
@@ -29,10 +30,15 @@ const (
 	ErrUnknownABI             = fault.Const("Unknown device ABI: %+v")
 )
 
+// LibraryType enumerates the possible GAPID dynamic libraries.
+type LibraryType int;
+const (
+	LibGraphicsSpy LibraryType = iota
```
Comment (ben-clayton) [inline core/app/layout/layout.go]: This PR needs gofmting. I would encourage you to try out VSCode with the go plugin - it's an awesome go editing environment and also automatically formats as you save.

Comment (pmuetschard) [inline core/app/layout/layout.go]: Done.

**Comment by pmuetschard** [comment]
I've run stuff through gofmt and fixed the linter squiggles from VS code.
I've uploaded a change to make tracing find the MSYS dlls, which has the side effect of fixing windows tracing from the command line without the ENV setup of 'do run'.

**Comment by AWoloszyn** [comment]
Just tried this out for Vulkan on Linux / Android. Everything is working :+1:

**Pull request merged**